corehq/apps/userreports/templates/userreports/paywall/activating_subscription.html

```diff
@@ -0,0 +1,22 @@
+{% extends "style/bootstrap3/base_section.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load compress %}
+
+{% block title %}{% trans "Report Builder" %}{% endblock %}
+
+{% block page_content %}
+    {% block main_column_content %}
+        <h1>
+        {% trans "Subscribing to Report Builder" %}
+        </h1>
+        <p>
+            {% blocktrans %}
+                Thanks for your interest in Report Builder!
+                One of our accounts teams will be in touch soon to discuss your reporting
+                needs and turn report builder on for your project. If you have any further
+                questions please contact us at support@dimagi.com
```

Comment:
we probably have this email in a setting.  Also seems like a good idea to avoid putting plaintext emails in the source code to avoid bots from finding them.